js/map.js

```diff
@@ -44,6 +44,7 @@
         var marker = new google.maps.Marker({
           position: f_evt.feature.getGeometry().get(),
           title: f_evt.feature.getProperty('name'),
+          pid: f_evt.feature.getProperty('pid'),
```

Comment:
Looks like this is being added to the [geojson_spec in the undora.module](https://github.com/discoverygarden/undora/pull/21/files#diff-0f062a97dae6178708de4e7d0149157fR156), and made use of here. I am worried that using this code when undora is not present (or, enabled), could end up passing NULL or blowing up the js entirely. Instead, even tho it is only one line, i would suggest copying this file into the themes /js directory, and implementing hook_js_alter() to replace js/map.js with the theme /js version, including this work, if it is present in the javascript array to be added to the page.

It is pretty custom stuff, so we shouldn't feel to bad about doing that.

Reply:
I did try things out without undora enabled and things did seem to work fine, but I agree that customizations shouldn't be merged into the core module. I'll close this out and add it to the theme 👍